ulid.go

```diff
@@ -276,6 +291,55 @@ func (id *ULID) UnmarshalText(v []byte) error {
 	return nil
 }
 
+// UnmarshalTextStrict is like UnmarshalText, but additionally validates that
+// the parsed ULID consists only of valid base32 characters. It is slightly
+// slower than UnmarshalText.
+//
+// ErrDataSize is returned if the len(v) is different from an encoded
+// ULID's length. Invalid encodings return ErrInvalidCharacters.
+func (id *ULID) UnmarshalTextStrict(v []byte) error {
```

Comment:
The reason for the naming `UnmarshalText` is that it implements the [text.Unmarshaler](https://golang.org/pkg/encoding/#TextUnmarshaler) interface. I don't see a reason to have this symmetry here.

Reply:
Makes sense, thanks.